Update sql-set-background-ddl.mdx (#508)
risingwavelabs/risingwave-docs: sql/commands/sql-set-background-ddl.mdx

  />
</CardGroup>

> Note: For sinks, when they're created under `background_ddl` mode, they are treated as `created`, since their downstreams will immediately start receiving the backfill data. As such we don't track their backfill progress via `SHOW JOBS`.

Reviewer: ```suggestion
<Note>For sinks created under `background_ddl` mode, RisingWave treats them as `created` because their downstreams immediately start receiving the backfill data. Therefore, `SHOW JOBS` does not track their backfill progress.</Note>
```
